### DISCUSSION TITLE
Android (and mobile in general) improvement

### DISCUSSION THREAD

**Discussion by Kuina-chan** [ppy/osu#23549] [Ideas]
There are a lot of changes to Android (and maybe the entire mobile platform) to perform a little bit better

Note: I'm using a Samsung S10+ to based my ideas on.

- Refresh rates should be cap on 30, 45 and 60fps (for older devices to live on). Right now the default VSync aka 60fps create a lot of spike for my device.
- Having option to lower the resolution of the game, this will reduce the GPU load
- GPU compatible with Android (I know this being developed)
- Reduce animation anywhere possible, especially the audio visualizer around the osu logo and some animation when putting the panels up and down (those on the top of the screen)

Other ideas that not related to performance:

- Showing the size of the deadzone while changing it.
- Turn off leaderboard when watching Autoplay

**Comment by bdach** [comment]
We are committed to not sacrificing the visual fidelity of the game just to make it work on mobile. The hope is that we can optimise further to the point that removing effects will not be required. Thanks for your understanding.

> Showing the size of the deadzone while changing it.

What "deadzone" is being referred to here?

**Reply by Kuina-chan** [reply]
![Screenshot_20230515_070522.png](https://github.com/ppy/osu/assets/75318520/b86bd9ac-3f4a-4710-859a-feabec538859)

I mean this. If I'm not mistaken, this setting allow you to resize the area for the two extra input.

**Reply by bdach** [reply]
> this setting allow you to resize the area for the two extra input

"Two extra input"? You mean joystick and gamepad, correct? How would would you expect this was shown?

**Reply by Theighlin** [reply]
By two extra inputs he likely means mobile touch input in osu standard

**Reply by bdach** [reply]
> By two extra inputs he likely means mobile touch input in osu standard

If that is what was meant then that is not what the deadzone setting does at all. The deadzone setting is for joysticks / gamepads / controllers.
